CryptoPkg/Library/BaseCryptLibMbedTls/Pk/CryptRsaPss.c

```diff
@@ -11,6 +11,9 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #include "InternalCryptLib.h"
 #include <mbedtls/rsa.h>
+#include <mbedtls/sha256.h>
+#include <mbedtls/sha512.h>
+#include <mbedtls/compat-2.x.h>

```

Comment:
delete <mbedtls/compat-2.x.h> and use Mbedtls3.0 apis

Reply:
Thanks. I have moved the  `<mbedtls/compat-2.x.h>`.